src/css/layout/_l-hero.scss

```diff
@@ -29,27 +36,15 @@
 			justify-content: center;
 		}
 	}
-
-	@supports #{$supports-grid} {
-		// Breakpoints
-		@include mappy-bp(lap-small) {
-			display: grid;
-			grid-template-rows: 1fr 1fr 1fr 3rem;
-			grid-template-areas:
-				". .    ."
-				". hero ."
-				". .    .";
-		}
-	}
 }
 
 
 .l-hero__promo {
 	margin-left: map-get($global-bleed, small);
 	margin-right: map-get($global-bleed, small);
 	position: relative;
-		top: 50%;
-	transform: perspective(1px) translateY(-60%); // HACK: Keeps content from rendering blurry
+	top: unset;
+	transform: perspective(1px) translateY(0); // HACK: Keeps content from rendering blurry
```

Comment:
These styles were contributing to overlap on short viewport.
I'm unfamiliar with this `transform` trick, but would it still be okay with the `translateY` is `0`? That way there's the rendering benefit without the positioning side effects.